src/App.jsx

```diff
@@ -0,0 +1,32 @@
+import {css, Global} from '@emotion/react';
+import Container from './components/Container';
+import Resume from './components/Resume';
+
+function App() {
+  return (
+    <Container>
+      <Global
+        styles={css`
```

Comment:
emotion에서 전역 css를 설정할 때 Global 컴포넌트를 사용하는데, 이 컴포넌트가 styles prop를 받는 것 같더라구요. 요 문서를 참고해서 작성했습니다!

https://emotion.sh/docs/globals

Reply:
오 정말 그러네요! 
```jsx
export let Global: React.AbstractComponent<GlobalProps> =
  /* #__PURE__ */ withEmotionCache((props: GlobalProps, cache) => {
    if (
      process.env.NODE_ENV !== 'production' &&
      !warnedAboutCssPropForGlobal && // check for className as well since the user is
      // probably using the custom createElement which
      // means it will be turned into a className prop
      // $FlowFixMe I don't really want to add it to the type since it shouldn't be used
      (props.className || props.css)
    ) {
      console.error(
        "It looks like you're using the css prop on Global, did you mean to use the styles prop instead?"
      )
      warnedAboutCssPropForGlobal = true
    }
```
https://github.com/emotion-js/emotion/blob/main/packages/react/src/global.js#L23-L37
에서도 css prop이나 className prop을 제한하고 있는 걸 확인했습니다